Scripts/Haptics/XRNodeHapticPulse.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace VRTK.Core.Haptics
+{
+    using UnityEngine;
+    using UnityEngine.XR;
+
+    /// <summary>
+    /// Creates a timed haptic pulse on a <see cref="XRNode"/> supported haptic device.
+    /// </summary>
+    public class XRNodeHapticPulse : HapticPulse
+    {
+        /// <summary>
+        /// The device to pulse.
+        /// </summary>
+        [Tooltip("The device to pulse.")]
+        public XRNode device = XRNode.LeftHand;
+        /// <summary>
+        /// The duration to pulse the device for.
+        /// </summary>
+        [Tooltip("The duration to pulse the device for.")]
+        public float duration = 0.005f;
+
+        /// <summary>
+        /// The haptic capabilities of the specified <see cref="XRNode"/>.
+        /// </summary>
+        protected HapticCapabilities caps = new HapticCapabilities();
+
+        /// <inheritdoc />
+        public override bool IsActive()
+        {
+            return base.IsActive();
+        }
+
+        /// <inheritdoc />
+        protected override void DoBegin()
+        {
+            Pulse(Intensity, duration, device);
```

Comment:
```suggestion
            Pulse(Intensity, duration);
```
The `node` is an instance variable, thus we don't need to pass it in.